webapp/src/pages/eventStats/components/EventStatsComponent.js

```diff
@@ -11,23 +11,47 @@ class EventStatsComponent extends Component {
 
     this.state = {
       loading: true,
+      buttonLoading: false,
       stats: null,
-      error: ""
+      error: "",
+      emailSendStatus: ""
     };
   }
 
   componentDidMount() {
     eventStatsService.getStats(DEFAULT_EVENT_ID).then(result=>{
       this.setState({
         loading: false,
+        buttonLoading: false,
         stats: result.stats,
-        error: result.error
+        error: result.error,
+        emailSendStatus: ""
```

Comment:
It's not necessary to specify the state items that don't change, but it doesn't do any harm either :)